cmd/config/node-density/node-density.yml

```diff
@@ -36,6 +36,12 @@ jobs:
     podWait: false
     waitWhenFinished: true
```

Comment:
We might need `iterations` specific keys here for churn to work.

Reply:
Yes, Both of these I believe. One to enable and the other to specify the number.
``` 
    namespacedIterations: {{.NAMESPACED_ITERATIONS}}
    iterationsPerNamespace: {{.ITERATIONS_PER_NAMESPACE}}
```